src/frontend/src/sass/_statusBlock.scss

```
@@ -0,0 +1,51 @@
+.dropdownSatusContainer {

```

Just noting the use of camelcased class names. As per our naming convention, the naming should be 
```suggestion
.dropdown-status__container {
```
and so on with the other names.

---

I think it might be:
.dropdown__status-container
block-name__elem-name_mod-name_mod-val
where dropdown is block name and status-container is the element. 

I feel there is still ambiguity with BEM convention. Maybe we need to do a CSS cleanup as part of our tech backlog to keep naming consistent?